internal/web/log.go

```diff
@@ -0,0 +1,46 @@
+package web
```

Comment:
Missing license header.

```suggestion
// Copyright 2021 tsuru authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package web
```